pkg/packetbrokeragent/agent_test.go

```diff
@@ -461,63 +488,129 @@ func TestForwarder(t *testing.T) {
 			return ttnpb.Empty, nil
 		}
 
-		res, err := gs.UpdateGateway(ctx, &ttnpb.UpdatePacketBrokerGatewayRequest{
-			Gateway: &ttnpb.PacketBrokerGateway{
-				Ids: &ttnpb.PacketBrokerGateway_GatewayIdentifiers{
-					GatewayId: "foo-gateway",
-					Eui:       types.EUI64{0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88}.Bytes(),
-				},
-				ContactInfo: []*ttnpb.ContactInfo{
-					{
-						ContactType:   ttnpb.ContactType_CONTACT_TYPE_OTHER,
-						ContactMethod: ttnpb.ContactMethod_CONTACT_METHOD_EMAIL,
-						Value:         "admin@example.com",
+		t.Run("Only contactInfo present", func(t *testing.T) { // nolint:paralleltest
```

Comment:
Check whether we need this test case as the `contact_info` field is not set in code

Reply:
Also inconsistent casing `contactInfo`; it's `contact_info` or `ContactInfo`